app-modules/alert/src/Observers/AlertObserver.php

```diff
@@ -17,4 +18,9 @@ public function created(Alert $alert): void
 
         AlertCreated::dispatch($alert);
     }
+
+    public function saved(Alert $alert): void
```

Comment:
Do we also want to listen for the `deleted` event?

Reply:
Yeah that is a good point, added.